### ISSUE TITLE
[bruteforce]impossible level, why account "locked" if $timenow > $timeout ?

### ISSUE THREAD

**Issue by raydonovan** [ethicalhack3r/DVWA#125]
Hello,

I don't understand the code above:

```
                // Calculate when the user would be allowed to login again
                $last_login = $row[ 'last_login' ];
                $last_login = strtotime( $last_login );
                $timeout    = strtotime( "{$last_login} +{$lockout_time} minutes" );
                $timenow    = strtotime( "now" );

                // Check to see if enough time has passed, if it hasn't locked the account
                if( $timenow > $timeout )
                        $account_locked = true;
```

Why is the current account set as "account_locked=true" if $timenow > $timeout ?
Isn't it suppose to be the opposite way ? 
Because if $timenow > $timeout, so the 15 minutes period before authenticate again passed, indeed the account should be set as account_locked=false and authentication come back ?

Regards

**Comment by digininja** [comment]
After long consideration, looks like the code never worked anyway, there were other bugs. Fixed in #203 

(Sorry for the delay!)

**Comment by mike-bailey** [comment]
Feel free to reopen if it turns out to not have solved.
